### PR TITLE
refactor: move footer info to top in error_exception.php

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
See https://github.com/codeigniter4/CodeIgniter4/pull/8616#issuecomment-1997478306
It is useful when reporting errors.

Before:
Screehshot 2024-03-15 9 18 50

After:
Screenshot 2024-03-15 11 57 39

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: app/Views/errors/html/debug.css [hunk 1]
```diff
@@ -65,13 +65,10 @@ p.lead {
     display: inline;
 }
 
-.footer {
+.environment {
     background: var(--dark-bg-color);
     color: var(--light-text-color);
-}
-.footer .container {
     border-top: 1px solid #e7e7e7;
```
Comment (datamweb) [inline app/Views/errors/html/debug.css:71]:
```suggestion
```
I think we don't need it.

Screenshot 2024-03-15 043000

Comment (kenjis) [reply app/Views/errors/html/debug.css]: Okay. Removed.

File: app/Views/errors/html/debug.css [hunk 2]
```diff
@@ -65,13 +64,9 @@ p.lead {
     display: inline;
 }
 
-.footer {
+.environment {
     background: var(--dark-bg-color);
     color: var(--light-text-color);
```
Comment (ddevsr) [inline app/Views/errors/html/debug.css:69]: Can you give space padding text to background? like a before fixed but change to proportional. To be honest i want in flying bottom like debug bar

**Review by ddevsr — CHANGES_REQUESTED** [review]

File: app/Views/errors/html/debug.css (hunk 2 above)
Comment (kenjis) [reply app/Views/errors/html/debug.css:69]: Added `padding: 0.2rem;`:
Screenshot 2024-03-15 11 57 39

**Review by lonnieezell — APPROVED** [review]

**Review by ddevsr — APPROVED** [review]

**Review by MGatner — APPROVED** [review]

**Review by datamweb — APPROVED** [review]
Thanks. LGTM!

**Pull request merged**